### DISCUSSION TITLE
C언어 기반 K-WASM 기능 구현이 완료되었습니다.

### DISCUSSION THREAD

**Discussion by Hyeopgeon-Lee** [K-PaaS-incubator/k-wasm-app#5] [Announcements]
현재 C언어 기반으로 K-WASM 앱을 구현하였으며, 초기 버전으로 예상치 못한 오류가 많이 발생할 수 있습니다.
발생되는 에러는 Issues 게시판에 올려주시면, 조치하도록 하겠습니다.

**Comment by chlwnstkd** [comment]
좋은 정보 감사합니다
